ddtrace/debugging/_function/discovery.py

```diff
@@ -9,7 +9,7 @@
 try:
     from collections.abc import Iterator
 except ImportError:
-    from collections import Iterator
+    from collections import Iterator  # type: ignore[attr-defined,no-redef]
```

Comment:
🤔 I wonder why `attr-defined` is needed here. Perhaps something to do with the import above?

Reply:
Yes, because in Python 3.10 there is no `collections.Iterator` anymore so mypy is saying that this is not going to work 😉